dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/ProcessDefinitionController.java

```diff
@@ -296,8 +296,13 @@ public Result queryProcessDefinitionVersions(@ApiIgnore @RequestAttribute(value
                                                  @RequestParam(value = "pageNo") int pageNo,
                                                  @RequestParam(value = "pageSize") int pageSize,
                                                  @RequestParam(value = "processDefinitionId") int processDefinitionId) {
-
-        Map<String, Object> result = processDefinitionVersionService.queryProcessDefinitionVersions(loginUser
+        logger.info("query process definition versions, login user {}, project name: {}, process define id: {}, list paging, pageNo: {}, pageSize: {}",
+                loginUser.getUserName(), projectName, processDefinitionId, pageNo, pageSize);
```

Comment:
![image](https://user-images.githubusercontent.com/4902714/109378110-d2e83800-790a-11eb-9178-7db1f4462f32.png)
TO fix the vulnerability  `Change this code to not log user-controlled data`, you can use `RegexUtils.escapeNRT(loginUser.getUserName())`. Otherwise, you cannot pass the SonarCloud Quality Gate Test.